cli/script/command-executor.ts

```diff
@@ -896,11 +896,14 @@ function getReactNativeProjectAppVersion(command: cli.IReleaseReactCommand, proj
             throw new Error(`The "CFBundleShortVersionString" key doesn't exist within the "${resolvedPlistFile}" file.`);
         }
     } else if (command.platform === "android") {
-        const buildGradlePath: string = path.join("android", "app", "build.gradle");
+        let buildGradlePath: string = path.join("android", "app", "build.gradle");
+        if (command.gradleFile) {
+            buildGradlePath = command.gradleFile;
+        }
         if (fileDoesNotExistOrIsDirectory(buildGradlePath)) {
-            throw new Error(`Unable to find the "build.gradle" file in your "android/app" directory.`);
+            throw new Error(`Unable to find gradle file "${buildGradlePath}"".`);
```

Comment:
Unable to find the gradle file at "${buildGradlePath}"

You also had an extra `"`